### DISCUSSION TITLE
Custom action to pass files with results from one Pick grep into glob parameter for next grep

### DISCUSSION THREAD

**Discussion by ghost** [nvim-mini/mini.nvim#1342] [Q&A]
I would like to grep my notes for #tag1 and then take the files for all selected results and use an action to pass them into the glob local_opt and grep those files for #tag2, but I'm a bit stuck on how to do that.

**Comment by echasnovski** [comment]
Here are relevant parts of docs which should help:
- [How to create custom action](https://github.com/echasnovski/mini.nvim/blob/b07157c51e52a1b004ec7959f9618e9bc24686d3/doc/mini-pick.txt#L590).
- [How to get items matching current query](https://github.com/echasnovski/mini.nvim/blob/b07157c51e52a1b004ec7959f9618e9bc24686d3/doc/mini-pick.txt#L1245).
- [Where to put custom globs in the `builtin.grep_live` picker](https://github.com/echasnovski/mini.nvim/blob/b07157c51e52a1b004ec7959f9618e9bc24686d3/doc/mini-pick.txt#L1123).

I think this should clarify the most difficult 'mini.pick' parts. If I understood correctly, the most difficult part left is to convert the matched items from the first picker into the globs of the second. Feel free to ask questions if nothing you tried worked.
